feat: make openai key not required (#192)
speakeasy-api/speakeasy: docs/suggest.md

## Details

The "suggest" command validates an OpenAPI spec and uses OpenAI's ChatGPT to suggest fixes to your spec.

> You can use the Speakeasy OpenAI key within our platform limits, or you can set your OpenAI API key in a OPENAI_API_KEY environment variable. You will also need to authenticate with the Speakeasy API,

Reviewer: ```suggestion
You can use the Speakeasy OpenAI key within our platform limits, or you may set your own using the OPENAI_API_KEY environment variable. You will also need to authenticate with the Speakeasy API,
```
